bookmarker for pinboard extension/SafariExtensionHandler.swift

```diff
@@ -18,6 +18,10 @@ class SafariExtensionHandler: SFSafariExtensionHandler {
                         SafariExtensionViewController.shared.descriptionTextField.stringValue = ""
                     }
                 }
+            case "openPopover" :
+                NSLog("Got 'openPopover' message")
+                // Ideally if I had access to SFSafariWindow I could do window.getToolbarItem and invoke 'showPopover' but only
+                // have access to SFSafariPage. So not sure if I can make this work.
```

Comment:
Haven't written code in Swift in a few years but (why) can't you instantiate a `SFSafariToolbarItem` and call the `showPopover()` method mentioned in the Developer Forum? https://developer.apple.com/documentation/safariservices/sfsafaritoolbaritem?language=objc#declarations

Reply:
I don't just need any `SFSafariToolbarItem`, I need the one that is part of our extension which should get provided through the development framework.  

Apple provides a framework with several hooks that allows us to implement logic or pass information between different parts of the application like between the script that gets embedded in every webpage and the extension. The new App Extension framework is still relatively new and it looks like some things are still missing, like straightforward support for keyboard shortcuts.

Let's see what answer I get to my [question in the Apple Developer Forum](https://forums.developer.apple.com/thread/115651).